README.md

```diff
@@ -81,6 +81,7 @@ Software that provides an API to transit data.
 Tools for transit agencies.
 
 - [Remix](http://getremix.com/) - A webapp that lets transit agencies easily plan routes.
+- [Next Train API](https://github.com/data-creative/next-train-api) - Serves any GTFS feed as a JSON API! Transit agencies and developers alike can deploy the open source code to their own Heroku server.
```

Comment:
How is this different from https://github.com/s2t2/next-train-api?  Generally I prefer to link to original repos instead of forks.

Reply:
Also, I think Next Train API is better suited to the "APIs" section.

Reply:
I prefer to keep this list as objective and non-marketing as possible.  Could you please change the `!` to a `.`?

Reply:
Finally, I prefer to keep this list as objective and non-marketing as possible, so I'd prefer to change the `!` to a `.`.

Reply:
The Data Creative location reflects more functional version, and where the app is maintained moving forward.

It is better suited for the "tools" section because it is not a functioning API, but rather a deployable tool to help people create their own API.

Changed the ! to a .

Reply:
Some of the other entries in the "APIs" section (OBA, OTP, Transitime) don't have centrally hosted versions either - they are tools to deploy your own API.  So, the Next Train API would fit in that section considering the other entries.

Reply:
>The Data Creative location reflects more functional version, and where the app is maintained moving forward.

So the s2t2 repos are essentially deprecated and unmaintained going forward?  If so, my preference would be that you contact Github and request that the Data Creative repos become the new root repos, changing the s2t2 repos to be forks of those repos.  Currently it's very confusing to see the two different repos, and most peoples instincts will be to go to the root repo as the most recent version.